src/Services/MeilisearchService.php

```diff
@@ -374,6 +375,11 @@ private static function resolveClass(object $object): string
         static $resolver;
 
         $resolver ??= (function () {
+            // Native lazy objects compatibility
+            if (PHP_VERSION_ID >= 80400 && class_exists(LegacyReflectionFields::class)) {
+                return fn ($object) => get_class($object);
```

Comment:
CS Fixer job fails here because it wants us to use `$object::class`, but then test jobs fail with:
```
PHP Fatal error:  Cannot use ::class with dynamic class name in /home/runner/work/meilisearch-symfony/meilisearch-symfony/src/Services/MeilisearchService.php on line 380
```